iosApp/templateIOS.xcodeproj/project.pbxproj

```diff
@@ -456,6 +456,14 @@
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.0;
+				OTHER_LDFLAGS = (
```

Comment:
Need to find out if there's any downfalls of keeping this here if we're not using sqldelight.